### PR TITLE
Feature/consumer publisher created

### PR DESCRIPTION
**Pull request by RK206**

### CODE REVIEW

File: cms/data/notifications.yml [hunk 1]
```diff
@@ -21,3 +21,22 @@ application:maned:ready:notify:
...
+    We receive hundreds of applications every month. We will review yours as soon as we can. You will receive an email from us when the review is complete or if we have questions.
+    If you need to make a change to your application, send an email to our Help desk: helpdesk@doaj.org. Please include your ISSN.
+    We cannot reply to emails requesting a status update for applications which are less than three months old.
+    If you write to us, check first that there is nothing in your Spam folder from us or one of our volunteers. Our volunteers may not be emailing from a DOAJ email address so you can check that their name is there in the url mentioned below {volunteers_url}
+
+    Yours faithfully,
+    The DOAJ Editorial Team
+
+    ***
+    This is an automated message."
```
Comment (richard-jones) [inline cms/data/notifications.yml:40]: No need to add the sign off here, that's added by the notification emailer.

As an aside, we may want to look at shortening the text here to feel more like a notification, will raise that separately.

File: cms/data/notifications.yml [hunk 2]
```diff
@@ -21,3 +21,22 @@ application:maned:ready:notify:
...
+  email_subject:
+    "Directory of Open Access Journals - your application to DOAJ has been received"
```
Comment (richard-jones) [inline cms/data/notifications.yml:42]: No need to prefix with Directory of Open Access Journals - this is done by the notifier

File: portality/constants.py [hunk 3]
```diff
@@ -18,7 +19,8 @@
     APPLICATION_STATUS_IN_PROGRESS,
     APPLICATION_STATUS_COMPLETED,
     APPLICATION_STATUS_ON_HOLD,
-    APPLICATION_STATUS_READY
+    APPLICATION_STATUS_READY,
+    APPLICATION_STATUS_CREATED
```
Comment (richard-jones) [inline portality/constants.py:23]: Not a new status, so shouldn't be here

File: portality/constants.py [hunk 4]
```diff
@@ -8,6 +8,7 @@
 APPLICATION_STATUS_COMPLETED = "completed"
 APPLICATION_STATUS_ON_HOLD = "on hold"
 APPLICATION_STATUS_READY = "ready"
+APPLICATION_STATUS_CREATED = "created"
```
Comment (richard-jones) [inline portality/constants.py:11]: This isn't a new status.  I think the confusion here is that this notification has been wired to a "created" status, when in fact what we want is an event for the creation of a new application.  The initial application status will be "pending" or whatever it is set to.

File: portality/events/consumers/application_publisher_created_notify.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+from portality.lib.seamless import SeamlessException
+
+
+class ApplicationPublisherCreatedNotify(EventConsumer):
+    ID = "application:publisher:created:notify"
+
+    @classmethod
+    def consumes(cls, event):
+        return event.id == constants.EVENT_APPLICATION_STATUS and \
+               event.context.get("status") == constants.APPLICATION_STATUS_CREATED
```
Comment (richard-jones) [inline portality/events/consumers/application_publisher_created_notify.py:17]: I think we should be cosnuming an `application:created` event here

File: portality/forms/application_processors.py [hunk 6]
```diff
@@ -258,12 +258,12 @@ def finalise(self, account, save_target=True, email_alert=True, id=None):
...
-                emails.send_received_email(self.target)
-            except app_email.EmailException as e:
-                self.add_alert(Messages.FORMS__APPLICATION_PROCESSORS__NEW_APPLICATION__FINALISE__USER_EMAIL_ERROR)
-                app.logger.exception(Messages.FORMS__APPLICATION_PROCESSORS__NEW_APPLICATION__FINALISE__LOG_EMAIL_ERROR)
+        # trigger a status change event
+        eventsSvc = DOAJ.eventsService()
+        eventsSvc.trigger(models.Event(constants.EVENT_APPLICATION_STATUS, None, {
+            "application": self.target.data,
+            "status": constants.APPLICATION_STATUS_CREATED
+        }))
```
Comment (richard-jones) [inline portality/forms/application_processors.py:266]: This should be

```
eventsSvc.trigger(models.Event(constants.EVENT_APPLICATION_CREATE, current_user.id, {
            "application": self.target.data
}))
```

**Review by richard-jones — CHANGES_REQUESTED** [review]
A bit of a mix up here on the event type.  I know that Rama is away so I may come back and fix this one up myself shortly

**Comment by richard-jones** [comment]
I have now made these changes, so @Steven-Eardley @amdomanska perhaps you could give it a review now?

**Review by Steven-Eardley — APPROVED** [review]

**Pull request merged**